index.html

```diff
@@ -126,6 +127,153 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
       </div>
     </section>
 
+    <section class="menu-section">
+      <div class="menu-columns">
+        <div>
+          <section>
+            <h4 class="heading-secondary heading-dark">Appetisers</h4>
+            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
```

Comment:
Usually we start sentence with a capital letter
```suggestion
            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="Decorative element">
```